hasweb/templates/index.html

```diff
@@ -43,12 +43,18 @@
   </div>
   <ul class="cardset row">
     {%- for etitle, edate, eurl, edesc in [
-        ('Introduction to Data Science in Python', 'Sat, Mar 12 – Sun, Mar 13, Bangalore', 'https://unnatixyz.talkfunnel.com/intro-to-data-science-in-python-1/',
+        ('Introduction to Data Science in R', 'Sat, Apr 2 – Sun, Apr 3, Bangalore', 'https://unnatixyz.talkfunnel.com/intro-to-data-science-in-r-1/',
           'Ever wondered what it takes to work in the field of Data Science?'),
         ('Rootconf 2016', 'Thu, Apr 14 – Fri, Apr 15, Bangalore', 'https://rootconf.talkfunnel.com/2016/',
           'On learning from failure in DevOps'),
+        ('Metarefresh 2016', 'Fri, May 13 – Sat, May 14, Bangalore', 'http://metarefresh.in/',
```

Comment:
We should not list events on the site until the CfP is open. There has to be a clear call to action.